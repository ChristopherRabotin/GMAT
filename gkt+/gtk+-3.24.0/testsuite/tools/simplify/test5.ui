<!-- placeholders are stripped -->
<interface>
  <object class="GtkWindow">
    <child>
      <object class="GtkBox">
        <child>
          <placeholder/>
        </child>
      </object>
    </child>
  </object>
</interface>
