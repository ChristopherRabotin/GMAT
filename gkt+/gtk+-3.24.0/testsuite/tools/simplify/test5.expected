<!-- placeholders are stripped -->
<interface>
  <object class="GtkWindow">
    <child>
      <object class="GtkBox">
      </object>
    </child>
  </object>
</interface>
